public/base-locales/pl.json

```diff
@@ -488,15 +490,64 @@
   "webhook_advanced_save_width": 2,
   "login_button": 5,
   "join_button": 5,
+  "weather_0": "Extreme",
+  "weather_1": "Clear",
+  "weather_2": "Rain",
+  "weather_3": "Partly Cloudy",
+  "weather_4": "Cloudy",
+  "weather_5": "Windy",
+  "weather_6": "Snow",
+  "weather_7": "Fog",
+  "poke_type_0": "Brak",
+  "poke_type_1": "Normal",
+  "poke_type_2": "Fighting",
+  "poke_type_3": "Flying",
+  "poke_type_4": "Poison",
+  "poke_type_5": "Ground",
+  "poke_type_6": "Rock",
+  "poke_type_7": "Bug",
+  "poke_type_8": "Ghost",
+  "poke_type_9": "Steel",
+  "poke_type_10": "Fire",
+  "poke_type_11": "Water",
+  "poke_type_12": "Grass",
+  "poke_type_13": "Electric",
+  "poke_type_14": "Psychic",
+  "poke_type_15": "Ice",
+  "poke_type_16": "Dragon",
+  "poke_type_17": "Dark",
+  "poke_type_18": "Fairy",
+  "quest_reward_12": "Mega Energia",
```

Comment:
Why are weather conditions and pokemon types here? Especially in English? None of these should be necessary.

Reply:
Cause they suck in polish and by default they're translated in that file with default translations, so this file would overwrite them. No one uses polish versions of types amd other unnecessary translations, it's just confusing and sounds stupid